app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render partial: 'reviews/errors' %>
+<%= form_for @review, :url => {:controller => 'reviews', :action => 'update'} do |f| %>
```

Comment:
generally you don't NEED to override the URL like this, the contents of `@review` will do all of this for you by looking at the name of the resource, whether it has an ID etc